config/routes.rb

```diff
@@ -81,6 +81,12 @@
   match 'care_plan_contacts/update', to: 'care_plan_contacts#update', via: :patch
   match 'care_plan_contacts/destroy', to: 'care_plan_contacts#destroy', via: :delete
 
+  match 'moment_templates', to: 'moment_templates#index', via: :get
+
+  match 'moment_templates/create', to: 'moment_templates#create', via: :post
+  match 'moment_templates/update', to: 'moment_templates#update', via: :patch
+  match 'moment_templates/destroy', to: 'moment_templates#destroy', via: :delete
+
```

Comment:
Could these also be shortened to:

```suggestion
  resources :moment_templates, only: [:index, :create, :update, :destroy]
```